hadoop/Dockerfile

```diff
@@ -74,27 +76,42 @@ COPY --chown=${STACKABLE_USER_UID}:0 hadoop/stackable/jmx /stackable/jmx
 RUN <<EOF
 cd "$(/stackable/patchable --images-repo-root=src checkout hadoop ${PRODUCT})"
 
+ORIGINAL_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+NEW_VERSION=${PRODUCT}-stackable${RELEASE}
+
+mvn versions:set -DnewVersion=${NEW_VERSION}
+
+# Since we skip building the hadoop-pipes module, we need to set the version to the original version so it can be pulled from Maven Central
+sed -e '/<artifactId>hadoop-pipes<\/artifactId>/,/<\/dependency>/ { s/<version>.*<\/version>/<version>'"$ORIGINAL_VERSION"'<\/version>/ }' -i hadoop-tools/hadoop-tools-dist/pom.xml
+
 # Create snapshot of the source code including custom patches
-tar -czf /stackable/hadoop-${PRODUCT}-src.tar.gz .
+tar -czf /stackable/hadoop-${NEW_VERSION}-src.tar.gz .
 
 mvn \
     --batch-mode \
     --no-transfer-progress \
-    clean package \
+    clean package install \
     -Pdist,native \
-    -pl '!hadoop-tools/hadoop-pipes,!hadoop-yarn-project,!hadoop-mapreduce-project,!hadoop-minicluster' \
```

Comment:
We delete these jars later on anyway, but I think these were omitted here to speed up the build. Do we need to re-include them?